src/main/java/de/zalando/zmon/dataservice/config/OpenTracingConfigProperties.java

```diff
@@ -0,0 +1,119 @@
+package de.zalando.zmon.dataservice.config;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import org.springframework.context.annotation.Configuration;
+
+public class OpenTracingConfigProperties {
+
+    private String tracingProvider = "noop";
+    private String serviceName = "zmon-data-service";
+
+    private String jaegerHost = "localhost";
```

Comment:
Can we move provider-specific configuration options into separate classes? (the classes can be defined within `OpenTracingConfigProperties` class)